actors/builtin/power/power_state.go

```diff
@@ -206,6 +212,12 @@ func (st *State) appendCronEvent(events *adt.Multimap, epoch abi.ChainEpoch, eve
 	return nil
 }
 
+func (st *State) updateSmoothedEstimate(currEpoch abi.ChainEpoch) {
+	delta := currEpoch - st.LastCronEpoch
```

Comment:
I'm a bit uncertain about this using LastCronEpoch to compute a delta without then updating it to the current epoch. This could be resolved by computing the delta in the caller, which is where the update then happens.